lib/net/ssh/config.rb

```diff
@@ -154,7 +154,10 @@ def load(path, host, settings = {}, base_dir = nil)
             # ProxyCommand and ProxyJump override each other so they need to be tracked togeather
             %w[proxyjump proxycommand].each do |proxy_key|
               if (proxy_value = settings.delete(proxy_key))
-                settings['proxy'] ||= [proxy_key, proxy_value]
+                # When ProxyJump is set to none explicity disable the jumphost
```

Comment:
Shouldn't we clear settings['proxy'] if proxy jump is none. I mean if one config has a proxy configured but a more specific sets it none then we should clear it, shouldn't we?

Reply:
Sorry I totally missed this! Let me fix that up